src/hooks/react-query/feature-requests/useFeatureRequests.js

```
@@ -0,0 +1,155 @@
+        ]);
+
+        if (error) {
+            console.error('Error voting feature request:', error);
+            throw new Error('Failed to vote');
+        }
+    }
+};
+
+// Hook to vote/unvote on a feature request
+export const useVoteOnFeatureRequest = () => {
+    const queryClient = useQueryClient();
```

ESLint

[error] 'queryClient' is assigned a value but never used.

(no-unused-vars)